subjects/java-piscine/Quest1/01-02-int-operation/README.md

```diff
@@ -0,0 +1,76 @@
+## IntOperations
+
+### Instructions
+
+Create a file `IntOperation.java`.
+
+Write a function `addTwoIntegers` that returns the sum of two integers passed as parameter.
+Write a function `substractTwoIntegers` that returns subtraction of two integers passed as parameter.
+Write a function `multiplyTwoIntegers` that returns the multiplication of two integers passed as parameter.
+Write a function `divideTwoIntegers` that returns the Euclidian division of two integers passed as parameter. We ignore the divide by 0 case.
+
+### Expected Functions
+```java
+public class IntOperation {
+    public static int addTwoIntegers(int a, int b) {
+        // your code here
+    }
+    public static int subtractTwoIntegers(int a, int b) {
+        // your code here
+    }
+    public static int multiplyTwoIntegers(int a, int b) {
+        // your code here
+    }
+    public static int divideTwoIntegers(int a, int b) {
+        // your code here
+    }
+}
+```
+
+# Usage
+
+Here is a possible ExerciseRunner.java to test your function : 
+```java
+public class ExerciseRunner {
+    public static void main(String[] args) {
+        System.out.println("Add");
+        System.out.println(IntOperation.addTwoIntegers(1, 2));
+        System.out.println(IntOperation.addTwoIntegers(15, 68));
+        
+        System.out.println("Subtract");
+        System.out.println(IntOperation.subtractTwoIntegers(4, 1));
+        System.out.println(IntOperation.subtractTwoIntegers(10, 23));
+        
+        System.out.println("Multiply");
+        System.out.println(IntOperation.multiplyTwoIntegers(3, 6));
+        System.out.println(IntOperation.multiplyTwoIntegers(12, 11));
+        
+        System.out.println("Divide");
+        System.out.println(IntOperation.divideTwoIntegers(8, 2));
+        System.out.println(IntOperation.divideTwoIntegers(13, 4));
+    }
+}
+```
+
+and its output :
```

Comment:
You should install prettier so that you can format the readmes and avoid extra spaces like in line:
32,39, 43, 47, 58,71, 75

and it helps with the format of the radme example:
![image](https://user-images.githubusercontent.com/53977751/232837392-63d6fc18-0a53-4a81-9dd8-d690a50c1820.png)
How it looks after 
![image](https://user-images.githubusercontent.com/53977751/232837486-ab254cc1-8c7a-4590-9b0d-ef7fe71b40ff.png)